.gitignore

```diff
@@ -72,6 +72,7 @@ target/
 
 # Jupyter Notebook
 .ipynb_checkpoints
+notebooks/*.ipynb
```

Comment:
we might actually *want* to have notebooks here. if you don't want to add yours specifically it is better to not add it in git.